src/llvm/lib/Target/MSP430/MSP430.td

```diff
@@ -40,6 +40,228 @@ def FeatureHWMultF5
 class Proc<string Name, list<SubtargetFeature> Features>
  : Processor<Name, NoItineraries, Features>;
 
+def : Proc<"msp430c111",      []>;
```

Comment:
So, there is no point of defining all the CPUs here since they do not change much compiler-wise. Please shrink down to families at least.

Reply:
Got rid of it.